### ISSUE TITLE
[GEN2] Darvester Core - Configuration Wizard

### ISSUE THREAD

**Issue by V3ntus** [V3ntus/darvester#50]
Guide the user in configuring Darvester Core. (token, etc.)

Will probably be stored as JSON (?) then parsed in the Python

**Comment by MyRoWARE** [comment]
Interested, can I take this for the Hacktoberfest?

**Comment by V3ntus** [comment]
@MyRoWARE Sure thing! Make sure to read over [CONTRIBUTING.md](https://github.com/darvester/darvester/blob/main/CONTRIBUTING.md) to get setup. I'll assign you

**Comment by MyRoWARE** [comment]
Thank you! I'll start working on it.

**Comment by MyRoWARE** [comment]
@V3ntus  just a quick question before I start working on this. I just want to check if I understand correctly. Should I make these changes on gen2 or dev branch? This: "To contribute to Darvester core instead, please point your PRs to the dev branch." kinda confused me now because of the title.

**Comment by V3ntus** [comment]
@MyRoWARE Yes, sorry about the confusion! Changes to the `gen2` branch should reflect `gen2` as the targeting branch in the PR. This is where you would make changes to the React.js code for example. Darvester core would be the `main`/`dev` branches hosting only the Python distribution of this code, the previous generation (versions 1.x) of Darvester

**Comment by MyRoWARE** [comment]
Thank you for explaining it to me.

**Comment by MyRoWARE** [comment]
I've just got Darvester running and I'm ready to start working on this task. But before I'll start I would like to get more info. I noticed there already is FirstRun.jsx which pretty much guides user for first time setup. So what Im going to do is adding one more step to first run where I ask user for token and other settings from core? I just want to ask if thats right or not.

**Comment by V3ntus** [comment]
Hey @MyRoWARE currently the FirstRun.jsx only sets up the Python venv for Darvester Core and base config needed for the UI to work on. This does not include Core configuration. To get more acquainted with what Darvester Core is and the other sections, check out Program Layout in #44.

The FirstRun should implement basic config options (most importantly the Discord user token to log with). The config for Darvester Core can be found here https://github.com/darvester/darvester/blob/main/cfg.py.template
